com.unity.ml-agents/Runtime/SideChannels/EnvironmentParametersChannel.cs

```diff
@@ -18,10 +19,12 @@ internal enum EnvironmentDataTypes
     /// </summary>
     internal class EnvironmentParametersChannel : SideChannel
     {
-        Dictionary<string, float> m_Parameters = new Dictionary<string, float>();
+        Dictionary<string, Func<float>> m_Parameters = new Dictionary<string, Func<float>>();
         Dictionary<string, Action<float>> m_RegisteredActions =
             new Dictionary<string, Action<float>>();
 
+        SamplerFactory m_SamplerFactory = new SamplerFactory(1);
```

Comment:
We might want to allow the user to set individual seeds for each sampler from Python

Reply:
I actually meant to call this out. This is the end goal, just haven't implemented it yet since it was annoying.

Reply:
seeding has been taken care of.